src/views/QuestionView.vue

```diff
@@ -216,24 +222,14 @@ export default {
           this.filters.countryFilter !== "en:world"
             ? this.filters.countryFilter
             : null,
-          count
+          count,
+          this.page
         )
         .then((result) => {
           this.remainingQuestionCount = result.data.count;
-          if (result.data.questions.length == 0) {
-            if (!this.questionBuffer.includes(NO_QUESTION_LEFT)) {
-              if (clean) {
-                this.questionBuffer = [NO_QUESTION_LEFT];
-              } else {
-                this.questionBuffer = [
-                  ...this.questionBuffer,
-                  NO_QUESTION_LEFT,
-                ];
-              }
-            }
-            return;
+          if (this.remainingQuestionCount / count + 1 < this.page) {
```

Comment:
To be easier to read, do the calculation of the maximal index once at the beginning. Latter in your code it will be easier to understand `lastAvailablePageIndex` than a division plus some rounding

```suggestion
	const lastAvailablePageIndex = 1 + Math.ceil(result.data.count / count);
          if (lastAvailablePageIndex < this.page) {
```